fortitude/src/rules/style/implicit_none.rs

```diff
@@ -0,0 +1,71 @@
+/// Defines rules that raise errors if implicit typing is in use.
+use crate::ast::FortitudeNode;
+use crate::rules::correctness::implicit_typing::{
+    child_is_implicit_none, implicit_statement_is_none,
+};
+use crate::settings::Settings;
+use crate::{AstRule, FromAstNode};
+use ruff_diagnostics::{AlwaysFixableViolation, Diagnostic, Fix};
+use ruff_macros::{derive_message_formats, ViolationMetadata};
+use ruff_source_file::SourceFile;
+use tree_sitter::Node;
+
+/// ## What it does
+/// Checks for unnecessary `implicit none` in module procedures
+///
+/// ## Why is this bad?
+/// If a module has 'implicit none' set, it is not necessary to set it in contained
+/// functions and subroutines (except when using interfaces).
+#[derive(ViolationMetadata)]
+pub struct SuperfluousImplicitNone {
```

Comment:
I just noticed, the code for `SuperfluousImplicitNone` also still exists in the correctness `implicit_typing` file on the current main branch after this PR has been merged.

Reply:
Nice catch! I'm surprised that didn't cause any issues. Feel free to remove it in a new PR if you have the time.